packages/poppler/build.sh

```diff
@@ -4,4 +4,4 @@ TERMUX_PKG_VERSION=0.47.0
 TERMUX_PKG_SRCURL=http://poppler.freedesktop.org/poppler-${TERMUX_PKG_VERSION}.tar.xz
 # libcairo and littlecms is used by pdftocairo:
 TERMUX_PKG_DEPENDS="fontconfig, libcairo, libpng, libjpeg-turbo, libtiff, littlecms, openjpeg"
-TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--disable-poppler-glib"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--disable-poppler-glib  --enable-xpdf-headers"
```

Comment:
I guess the old xpdf headers are needed by texlive? Could you add a comment above this field stating that?